locale/fi/LC_MESSAGES/django.po

```diff
@@ -579,8 +580,9 @@ msgid "Reservation purpose"
 msgstr "Varaustarkoitukset"
 
 #: applications/admin/application_section/form.py
-#: merchants/admin/payment_order.py reservation_units/models/introduction.py
+#: reservation_units/models/introduction.py
 #: reservation_units/models/reservation_unit_pricing.py
+#: tilavarauspalvelu/admin/payment_product/admin.py
```

Comment:
cf. https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/1343#discussion_r1766203958

Reply:
Fixed.